devtoolsNG/xen/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright (c) 2020 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import './elements/inspector/data-explorer.js';
+
+const {logElt, opsByStoreElt, de} = window; // extract DOM elements
+
+de.object = {Raw: true, RawStoreMessage: true, StoreSyncMessage: true, StoreMessage: true};
+de.addEventListener('object-change', e => {
+  console.log(e);
+});
+
+const dom = (tag, props, parent) => parent.appendChild(Object.assign(document.createElement(tag), props));
+const createUi = (object, container) => dom('data-explorer', {object, expand: true}, container || logElt);
+
+const entries = [];
+
+const addRawEntry = function(raw) {
+  const entry = processRawEntry(raw);
+  entries.push(entry);
+  createUi({msg: entry}, logElt);
+  populateCollatedStoreEntries();
+};
+
+const processRawEntry = (rawEntry) => {
+  let data = rawEntry;
+  try {
+    data = JSON.parse(rawEntry);
+  } catch (x) {
+    //
```

Comment:
Empty comment?

Reply:
Empty catch blocks disallowed by linter. Do you have a preferred format or other solution?